docs/getting-started/quickstart-install.md

```diff
@@ -1,120 +1,7 @@
-# Install the Knative CLI
-
 --8<-- "install-kn.md"
```

Comment:
This page appears to be titled "Install the Knative CLI". This is a bit misleading because, if you follow all the instructions on the page, you will also be installing Knative using the quickstart plugin. 

Could we match the title of this page to the TOC, or at least have something that includes words like "install Knative" and/or "quickstart".